src/main/scala/io/sdkman/changelogs/MuleFlowDiagramsMigrations.scala

```diff
@@ -0,0 +1,24 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "027")
+class MuleFlowDiagramsMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_muleflowdiagrams_candidate",
+    author = "manikmagar"
+  )
+  def migration001(implicit db: MongoDatabase): Candidate = {
+    Candidate(
+      candidate = "muleflowdiagrams",
```

Comment:
I'm concerned that this is too verbose for our users. Could `mfd` or `mulefd` be better? This will be used as the candidate identifier that our users use on the CLI, the name will still be the same in all other contexts, listings etc.

Reply:
Sure, sounds good. I will change it to `mulefd`.